.github/workflows/tests.yml

```diff
@@ -0,0 +1,25 @@
+name: Run Tests
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v2
+
+    - name: Setup .NET
+      uses: actions/setup-dotnet@v2
+      with:
+        dotnet-version: '7.0'
+
+    - name: Restore dependencies
+      run: dotnet restore
+
+    - name: Run tests
+      run: dotnet test --filter FullyQualifiedName!~Bots
```

Comment:
Test for Bots should be handled locally since they are heavy, time consuming and sometimes random. The tests that need to pass on Master are from the ModuleTests and Tests projects.